torchbenchmark/models/unet1d/metadata.yml

```diff
@@ -0,0 +1,8 @@
+devices:
+  NVIDIA A100-SXM4-40GB:
+    eval_batch_size: 64
+eval_benchmark: true
+eval_deterministic: false
+eval_nograd: true
+train_benchmark: true
```

Comment:
Does this model support train test?

Reply:
Not yet. This should be a mistake. Thanks for pointing out.